android/src/main/java/com/brentvatne/exoplayer/FullScreenPlayerView.kt

```diff
@@ -60,10 +60,6 @@ class FullScreenPlayerView(
     init {
         setContentView(containerView, generateDefaultLayoutParams())
     }
-    override fun onBackPressed() {
-        super.onBackPressed()
-        onBackPressedCallback.handleOnBackPressed()
-    }
```

Comment:
Why removing this? (just for my knowledge)

Reply:
If the user presses the back button, the 'onStop' method will eventually be called as the activity transitions to a stopped state. Therefore, we can call the logic for backButtonPressed inside the 'onStop' function.